getstarted.rst

```diff
@@ -80,7 +96,8 @@ This part is simple. With the virtual environment active (which it should alread
 
 .. code-block:: bash
 
-    $ run_nio
+    cd ~/nio/projects/getting_started
+    run_nio
```

Comment:
Yeah I think it would be nice to add but we don't have it yet.

I think I would maybe call it `nio server`. I've been learning a little bit of rails recently. I like the way they do things and they use `rails server`.